src/content/api/module-methods.mdx

```diff
@@ -172,7 +172,7 @@ T> Note that `webpackInclude` and `webpackExclude` options do not interfere with
 The goal of CommonJS is to specify an ecosystem for JavaScript outside the browser. The following CommonJS methods are supported by webpack:
 
 
-### `require`
+### require
```

Comment:
Don't see the need to make it MDX.

Reply:
See this pull request https://github.com/webpack/webpack.js.org/pull/4412, with mdx we can customize the link with `Link` from `react-router` which would improve the page performance a lot because users don't have to load the full page again clicking on a link.

Reply:
Right :)